### DISCUSSION TITLE
Como puedo manipular los ion-buttons

### DISCUSSION THREAD

**Discussion by PedroRuizAguilar** [Aparking/AparKing_Backend#102] [Q&A]
Me gustaria saber como hacer para que los ion-buttons sean redonditos, ya que pienso que si fueran más redondos serían menos agresivos a la vista, como ha hecho @laurolmer , adjunto imagen de mis botones cuadrados
![imagen_2024-03-08_180653968](https://github.com/Aparking/AparKing_Backend/assets/72878120/e77f94eb-196a-4746-99eb-8b7356118093)

**Comment by laurolmer** [comment]
Peter dentro de tu etiqueta " ion-button " mete " shape="round" " y ya se ponen redonditos. 
Aquí te dejo un ejemplo crac:
<ion-button [routerLink]="'/garages/' + garage.id" color="secondary" shape="round" size="small">Editar</ion-button>
